django/db/models.py

```diff
@@ -431,18 +428,15 @@ def get_snippet_def(cls):
 
 class SmallMoleculeBatch(models.Model):
     smallmolecule           = models.ForeignKey('SmallMolecule')
-    facility_batch_id       = _CHAR(max_length=_BATCH_ID_LENGTH, **_NOTNULLSTR)
+    facility_batch_id       = _TEXT(max_length=_BATCH_ID_LENGTH, **_NOTNULLSTR)
     provider                = _TEXT(**_NULLOKSTR)
-    provider_catalog_id     = _CHAR(max_length=64, **_NULLOKSTR)
-    provider_sample_id      = _CHAR(max_length=35, **_NULLOKSTR)
+    provider_catalog_id     = _TEXT(**_NULLOKSTR)
+    provider_sample_id      = _TEXT(**_NULLOKSTR)
     chemical_synthesis_reference = _TEXT(**_NULLOKSTR)
     purity                  = _TEXT(**_NULLOKSTR)
     purity_method           = _TEXT(**_NULLOKSTR)
     aqueous_solubility      = models.DecimalField(max_digits=4, decimal_places=2, null=True)
-    aqueous_solubility_unit = models.CharField(null=True,
-                                               max_length=2,
-                                      choices=CONCENTRATION_WEIGHT_VOLUME_CHOICES,
-                                      default=CONCENTRATION_MGML)
+    aqueous_solubility_unit = models.TextField(null=True,default=CONCENTRATION_MGML)
```

Comment:
I realize default of mg/mL existed before this change, but I want to point out that I think it's dangerous to have any default here. The curator should have to explicitly specify the units and forgetting that should be an error rather than potentially incorrect metadata.